package.json

```diff
@@ -74,6 +74,7 @@
     "@canonical/cookie-policy": "2.1.0",
     "@canonical/global-nav": "2.4.1",
     "@canonical/latest-news": "1.0.3",
+    "moment": "^2.27.0",
```

Comment:
Can you pin the dependency? Although I think https://date-fns.org/ would be better for this as it's more modular. It's got a similar API to moment but you can just import the functions you need rather than the whole library.